### PR TITLE
Update conodictor to v2.3.1

### PR DESCRIPTION
**Pull request by Ebedthan**
Hello,
This PR upgrade the version of the conodictor. Thanks for your works!
Regards,

### CODE REVIEW

**Comment by osallou** [comment]
Hi, thanks for contrib. As shown in checks build failed due to missing labels or invalid labels.

**Comment by Ebedthan** [comment]
I think the build has now passed without an issue.

File: conodictor/2.3.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# base image
+FROM ubuntu:20.04
+
+# metadata
+LABEL base_image="ubuntu:20.04"
+LABEL version="2"
```
Comment (osallou) [inline conodictor/2.3.1/Dockerfile]: Version is 1 here as it is first version of this dockerfile.

Version is a way to track changes on a specific dockerfile (upstream is the same but we patch build rule for example)

**Review by osallou — CHANGES_REQUESTED** [review]
Thanks, just a final update on version label and it will be good for merge, thanks for your efforts

**Comment by osallou** [comment]
Merged, thanks for your work, container should be built in a moment after ci on master branch

**Pull request merged**